roscpp_azure_iothub/CMakeLists.txt

```diff
@@ -50,7 +45,11 @@ find_package(azure_iot_sdks CONFIG)
 if(azure_iot_sdks_FOUND)
   message(STATUS "azure-iot-sdk-c is found - ${IOTHUB_CLIENT_INCLUDES}")
 elseif(EXISTS "/usr/include/azureiot")
-  set(IOTHUB_CLIENT_INCLUDES "/usr/include/azureiot" "/usr/include/azureiot/inc")
+  if(MSVC)
```

Comment:
MSVC should be false if there is a /usr directory. (/usr/include only exists on Linux)